components/shared/layout/RightSidebar.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 const RightSidebar = () => {
   return (
-    <section className="sidebar-right border-l-[1.5px] text-white-100">
+    <section className="max-sm:display:none w-72 border-l-[1.5px] bg-black-800 text-white-100 max-lg:hidden">
```

Comment:
```suggestion
    <section className=" w-72 border-l-[1.5px] bg-black-800 text-white-100 max-lg:hidden">
```

Check `hidden` is already the tailwind class for
```css

display: none;
```

https://tailwindcss.com/docs/display

Reply:
removed the extra class